src/workerd/api/sql.h

```diff
@@ -28,12 +28,16 @@ class SqlStorage final: public jsg::Object, private SqliteDatabase::Regulator {
   jsg::Ref<Statement> prepare(jsg::Lock& js, kj::String query);
 
   double getDatabaseSize();
+  double getRowsRead();
```

Comment:
I agree that `Cursor`-level counters are better for single statements, but the database-level counters are good for finding IO hogs in an existing application. You can instrument your fetch handler to log IO at various points, you can build a flame graph like for CPU usage, and so on.

It could be that the typical sorts of things running in workerd are too small for that to be useful; I don't know.

I can move the counters to `Cursor` for now; if we find that we want database-level counters later on, they're not too hard to add. Does that sound reasonable?